packages/create-gatsby/src/index.ts

```diff
@@ -304,4 +335,14 @@ ${c.bold(`Thanks! Here's what we'll now do:`)}
   console.log(`See all commands at\n
   ${c.blueBright(`https://www.gatsbyjs.com/docs/gatsby-cli/`)}
   `)
+
+  trackCli(`CREATE_GATSBY_SUCCESS`)
 }
+
+process.on(`exit`, exitCode => {
+  trackCli(`CREATE_GATSBY_END`, { valueInteger: exitCode })
```

Comment:
let's use the `exitCode` here
```suggestion
  trackCli(`CREATE_GATSBY_END`, { exitCode })
```